### PR TITLE
add native notify docs

### PR DESCRIPTION
**Pull request by siovanus**

### CODE REVIEW

**Comment by tanZiWen** [comment]
add english version support

**Review by Honglei-Cong — CHANGES_REQUESTED** [review]
why Chinese version?

File: docs/specifications/native_contract/authnotify.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  ]
+}
+```
+
+
+* AssignFuncsToRole
+
+Assign auth of function to role
+
+Notify格式
```
Comment (Honglei-Cong) [inline docs/specifications/native_contract/authnotify.md]: "格式"?

File: docs/specifications/native_contract/authnotify.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+      ]
+    }
+  ]
+}
+```
+
+
+* AssignFuncsToRole
+
+Assign auth of function to role
```
Comment (Honglei-Cong) [inline docs/specifications/native_contract/authnotify.md]: "auth" = authentication?

Comment (Honglei-Cong) [reply docs/specifications/native_contract/authnotify.md]: Please fix all abbr. first.

**Review by Honglei-Cong — CHANGES_REQUESTED** [review]

File: docs/specifications/native_contract/authnotify.md (hunk 1 above)
Comment (jonatanblockchain) [reply docs/specifications/native_contract/authnotify.md]: > "格式"?

Same here:
../native_contract/authnotify.md
../native_contract/ongnotify.md
../native_contract/ontidnotify.md
../native_contract/ontnotify.md

**Review by laizy — APPROVED** [review]

**Pull request merged**